src/authentication/types.ts

```diff
@@ -2,14 +2,17 @@ export const authServerBaseUrlInjectSymbol = Symbol('auth-server-base-url');
 export const gitlabRootPasswordInjectSymbol = Symbol('gitlab-root-password');
 export const jwtOptionsInjectSymbol = Symbol('token-verify-options');
 
+export const subEmailClaimKey: 'https://sub_email' = 'https://sub_email';
+export const teamTokenClaimKey: 'https://team_token' = 'https://team_token';
```

Comment:
Could we either remove the `https://` part or then use it correctly, i.e. prepend the URL with our domain to namespace it to us? I'd prefer the first option since we'll be the only ones using these tokens so there's no risk of claim name conflict.

Also, aren't the type definitions superfluous?